src/index.js

```diff
@@ -14,6 +14,7 @@ const PeerInfo = require('peer-info')
 const PeerBook = require('peer-book')
 const mafmt = require('mafmt')
 const multiaddr = require('multiaddr')
+const Circuit = require('libp2p-circuit')
```

Comment:
This should be added in swarm, a feature that gets enabled, just like `connection.reuse`